pom.xml

```diff
@@ -375,7 +375,7 @@
         <version.io.jaegertracing>1.6.0</version.io.jaegertracing>
         <version.io.micrometer>1.9.3</version.io.micrometer>
         <version.io.netty>4.1.84.Final</version.io.netty>
-        <version.io.opentelemetry.opentelemetry>1.12.0</version.io.opentelemetry.opentelemetry>
+        <version.io.opentelemetry.opentelemetry>1.20.1</version.io.opentelemetry.opentelemetry>
```

Comment:
The ```1.21.0``` is available - does it make sense to upgrade for the release?

Reply:
Component update JIRA here: https://issues.redhat.com/browse/WFLY-17653